webcheck/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.9'
```

Comment:
Let's change this to version 3.7 to ensure compatibility with past OS versions. It should be fine as-is, but I'd like to err on the side of caution.

FYI, this will change in our upcoming apps refactor, but for now let's always use 3.7/

Reply:
Understood! :) I will take this into account in the future.